example/coffee/Example.react.coffee

```diff
@@ -0,0 +1,22 @@
+React = require('react')
+Icon = React.createFactory(require('../../index'))
+{div, h2} = React.DOM
+
+Example = React.createClass
+  displayName: 'Example'
+
+  render: ->
+    div null,
+      div className: 'example',
+        h2 null, 'No styles'
+        Icon(name: 'icon-clock')
```

Comment:
@tecnobrat I can't think of a good way to establish a prefix like `icon-`, and the more I think about it, the less of a good idea it seems.